public/js/actions/sources.js

```diff
@@ -184,6 +184,25 @@ function togglePrettyPrint(id) {
   };
 }
 
+function loadOriginalSourceText(source, getState, dispatch) {
+  return Task.spawn(function* () {
+    const generatedSource = getSource(
+      getState(),
+      getGeneratedSourceId(source)
+    );
+
+    const generatedSourceText = yield dispatch(
+      loadSourceText(generatedSource.toJS())
```

Comment:
I'm probably missing something here but why do we need to fetch the source at all if we just want to original source, and we already have that from the sourcemap?